ci/OWNERS

```diff
@@ -0,0 +1,2 @@
+labels: 
+  - "A: maintenance"
```

Comment:
I recommend using `A/thing` - more concise, leaves room for hierarchy if you decide later you want that, and is parseable as if it were a file-path via various stdlib support.

Reply:
Hmmm hierarchy is nice - but I find the `/` notation harder to read at a glance 😕 

What would be advantage of the latter?